### PR TITLE
Add nodetags

### PR DESCRIPTION
**Pull request by paurosello**
### What this PR does / why we need it

Add tags that are node specific, not for all resources

### Checklist

- [x] Updated CHANGELOG.md.

### Trigger E2E tests

/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-aws/templates/_machine_pools.tpl [hunk 1]
```diff
@@ -13,6 +13,7 @@ spec:
   additionalTags:
     k8s.io/cluster-autoscaler/enabled: "true"
     giantswarm.io/machinepool: {{ $name }}
+    {{- if $.Values.global.providerSpecific.additionalNodeTags -}}{{- toYaml $.Values.global.providerSpecific.additionalNodeTags | nindent 4 }}{{- end}}
```
Comment (Gacko) [inline helm/cluster-aws/templates/_machine_pools.tpl:16]:
```suggestion
    {{- if $.Values.global.providerSpecific.additionalNodeTags }}
    {{- toYaml $.Values.global.providerSpecific.additionalNodeTags | nindent 4 }}
    {{- end}}
```

File: helm/cluster-aws/templates/_control_plane.tpl [hunk 2]
```diff
@@ -5,6 +5,8 @@ Any changes to this will trigger the resource to be recreated rather than attemp
 */}}
 {{- define "controlplane-awsmachinetemplate-spec" -}}
 {{- include "ami" $ }}
+additionalTags:
+  {{- if $.Values.global.providerSpecific.additionalNodeTags -}}{{- toYaml $.Values.global.providerSpecific.additionalNodeTags | nindent 2 }}{{- end}}
```
Comment (Gacko) [inline helm/cluster-aws/templates/_control_plane.tpl:9]:
```suggestion
{{- if $.Values.global.providerSpecific.additionalNodeTags }}
additionalTags: {{ toYaml $.Values.global.providerSpecific.additionalNodeTags | nindent 2 }}
{{- end}}
```

**Review by fiunchinho — CHANGES_REQUESTED** [review]
This should be mentioned in the changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Add `global.providerSpecific.additionalNodeTags`. Field used to specify tags applied to nodes only.
```
Comment (fiunchinho) [inline CHANGELOG.md:10]:
```suggestion
### Added

- Add `global.providerSpecific.additionalNodeTags`. Field used to specify tags applied to nodes only.
```

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**